worlds/kh1/__init__.py

```diff
@@ -0,0 +1,315 @@
+                    slot_data["keyblade_stats"] = slot_data["keyblade_stats"] + str(str_bonus) + "," + str(mp_bonus) + ","
+            slot_data["keyblade_stats"] = slot_data["keyblade_stats"][:-1]
+        if self.options.donald_death_link:
+            slot_data["donalddl"] = ""
+        if self.options.goofy_death_link:
+            slot_data["goofydl"] = ""
+        if self.options.keyblades_unlock_chests:
+            slot_data["chestslocked"] = ""
+        else:
+            slot_data["chestsunlocked"] = ""
+        if self.options.interact_in_battle:
+            slot_data["interactinbattle"] = ""
+        return slot_data
+    
+    def create_item(self, name: str) -> KH1Item:
+        data = item_table[name]
+        return KH1Item(name, data.classification, data.code, self.player)
+
+    def create_event(self, name: str) -> KH1Item:
+        data = event_item_table[name]
+        return KH1Item(name, data.classification, data.code, self.player)
+
+    def set_rules(self):
+        set_rules(self.multiworld, self.player, self.options, self.determine_reports_required_to_open_end_of_the_world(), self.determine_reports_required_to_open_final_rest_door(), self.options.final_rest_door.current_key)
+
+    def create_regions(self):
+        create_regions(self.multiworld, self.player, self.options)
+    
+    def get_numbers_of_reports_to_consider(self) -> int:
+        numbers_to_consider = []
+        if self.options.end_of_the_world_unlock.current_key == "reports":
+            numbers_to_consider.append(self.options.required_reports_eotw)
+        if self.options.final_rest_door.current_key == "reports":
+            numbers_to_consider.append(self.options.required_reports_door)
+        if self.options.final_rest_door.current_key == "reports" or self.options.end_of_the_world_unlock.current_key == "reports":
+            numbers_to_consider.append(self.options.reports_in_pool)
+        numbers_to_consider.sort()
+        return numbers_to_consider
+    
+    def determine_reports_in_pool(self):
+        numbers_to_consider = self.get_numbers_of_reports_to_consider()
+        if len(numbers_to_consider) > 0:
+            return max(numbers_to_consider)
+        else:
+            return 0
+    
+    def determine_reports_required_to_open_end_of_the_world(self):
+        if self.options.end_of_the_world_unlock.current_key == "reports":
+            numbers_to_consider = self.get_numbers_of_reports_to_consider()
+            if len(numbers_to_consider) > 0:
+                return numbers_to_consider[0]
+        return 14
+    
+    def determine_reports_required_to_open_final_rest_door(self):
+        if self.options.final_rest_door.current_key == "reports":
+            numbers_to_consider = self.get_numbers_of_reports_to_consider()
+            if len(numbers_to_consider) == 3:
+                return numbers_to_consider[1]
+            elif len(numbers_to_consider) == 2:
+                return numbers_to_consider[0]
+        return 14
```

Comment:
```suggestion
        return 14
        
```
New line at end of file